crates/polars-time/src/windows/duration.rs

```diff
@@ -426,17 +400,20 @@ impl Duration {
         let minute = ts.minute();
         let sec = ts.second();
         let nsec = ts.nanosecond();
-        new_datetime(year, month as u32, day, hour, minute, sec, nsec).ok_or(
-            polars_err!(
-                ComputeError: format!(
-                    "cannot advance '{}' by {} month(s). \
-                        If you were trying to get the last day of each month, you may want to try `.dt.month_end` \
-                        or append \"_saturating\" to your duration string.",
-                        ts,
-                        if negative {-n_months} else {n_months}
-                )
-            ),
-        )
+        new_datetime(year, month as u32, day, hour, minute, sec, nsec).ok_or(polars_err!(
+            ComputeError: format!(
+                "cannot advance '{}' by {} month(s), datetime {}-{}-{}T{}:{}:{}.{} does not exist.",
+                    ts,
+                    if negative {-n_months} else {n_months},
+                    year,
+                    month,
+                    day,
+                    hour,
+                    minute,
+                    sec,
+                    nsec
+            )
+        ))
```

Comment:
Can this still happen?

Reply:
I wouldn't expect it to, have changed to `expect`, thanks